olca-app/src/org/openlca/app/db/tables/UnitItemTable.java

```diff
@@ -0,0 +1,156 @@
+package org.openlca.app.db.tables;
+
+import org.eclipse.jface.viewers.ITableLabelProvider;
+import org.eclipse.jface.viewers.LabelProvider;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.ui.IEditorInput;
+import org.eclipse.ui.IEditorSite;
+import org.eclipse.ui.PartInitException;
+import org.eclipse.ui.forms.IManagedForm;
+import org.eclipse.ui.forms.editor.FormPage;
+import org.openlca.app.M;
+import org.openlca.app.db.Database;
+import org.openlca.app.editors.Editors;
+import org.openlca.app.editors.SimpleEditorInput;
+import org.openlca.app.editors.SimpleFormEditor;
+import org.openlca.app.rcp.images.Images;
+import org.openlca.app.util.ErrorReporter;
+import org.openlca.app.util.Labels;
+import org.openlca.app.util.MsgBox;
+import org.openlca.app.util.UI;
+import org.openlca.app.viewers.Viewers;
+import org.openlca.app.viewers.tables.Tables;
+import org.openlca.core.model.ModelType;
+import org.openlca.core.model.Unit;
+import org.openlca.core.model.UnitGroup;
+import org.openlca.util.Strings;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class UnitItemTable extends SimpleFormEditor {
```

Comment:
better just name it `UnitTable`; the `Item` thing is just an internal detail and should not exposed to the name

Reply:
Make sense, fixed!